contracts/verifiers/UniversalVerifierMultiQuery.sol

```diff
@@ -0,0 +1,801 @@
+    }
+
+    // solhint-disable-next-line
+    // keccak256(abi.encode(uint256(keccak256("iden3.storage.UniversalVerifierMultiQuery")) -1 )) & ~bytes32(uint256(0xff));
+    bytes32 internal constant UniversalVerifierMultiQueryStorageLocation =
+        0x4235c64ddf027641dd9aa586d246e4cc3acfcb3f7016a8aa68f7ac21d38b3b00;
+
+    /**
+     * @dev Event emitted upon submitting a request
+     */
+    event ResponseSubmitted(uint256 indexed requestId, address indexed caller);
+
+    /**
+     * @dev Event emitted upon adding a request
+     */
+    event RequestSet(
+        uint256 indexed requestId,
+        address indexed requestOwner,
+        string metadata,
+        address validator,
+        bytes params
+    );
+
+    /**
+     * @dev Event emitted upon adding an auth request by the owner
+     */
+    event AuthRequestSet(
+        uint256 indexed requestId,
+        address indexed requestOwner,
+        string metadata,
+        address validator,
+        bytes params
+    );
+
+    /**
+     * @dev Event emitted upon updating a request
+     */
+    event RequestUpdate(
+        uint256 indexed requestId,
+        address indexed requestOwner,
+        string metadata,
+        address validator,
+        bytes params
+    );
+
+    /**
+     * @dev Event emitted upon adding a query
+     */
+    event QuerySet(uint256 indexed queryId, uint256[] requestIds);
+
+    /**
+     * @dev Event emitted upon updating a query
+     */
+    event QueryUpdate(uint256 indexed queryId, uint256[] requestIds);
+
+    /**
+     * @dev Modifier to check if the request exists
+     */
+    modifier checkRequestExistence(uint256 requestId, bool existence) {
+        if (existence) {
+            require(requestIdExists(requestId), "request id or auth request id doesn't exist");
+        } else {
+            require(!requestIdExists(requestId), "request id or auth request id already exists");
+        }
+        _;
+    }
+
+    /**
+     * @dev Modifier to check if the request exists
+     */
+    modifier checkRequestGroupExistence(uint256 groupId, bool existence) {
+        if (existence) {
+            require(groupIdExists(groupId), "group id doesn't exist");
+        } else {
+            require(!groupIdExists(groupId), "group id already exists");
+        }
+        _;
+    }
+
+    /**
+     * @dev Modifier to check if the query exists
+     */
+    modifier checkQueryExistence(uint256 queryId, bool existence) {
+        if (existence) {
+            require(queryIdExists(queryId), "query id doesn't exist");
+        } else {
+            require(!queryIdExists(queryId), "query id already exists");
+        }
+        _;
+    }
+
+    /**
+     * @dev Modifier to check if the validator is whitelisted
+     */
+    modifier onlyWhitelistedValidator(IRequestValidator validator) {
+        require(isWhitelistedValidator(validator), "Validator is not whitelisted");
+        _;
```

Comment:
Remove the whitelisting functionality please.
It is something that should be outside of the basic functionality (the same as now)